recipes/libelfin/all/conandata.yml

```
@@ -6,3 +6,5 @@ patches:
   "0.3":
     - patch_file: "patches/commit-9d0db16d0a0b3c4f8aaa60a3e4dab295df34b6b2.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/const-fix.patch"
+      base_path: "source_subfolder"
```

```suggestion
      base_path: "source_subfolder"
      # patch_source: https://github.com/aclements/libelfin/pull/54
```

When can document the patch with, at least, the URL where it comes from. [Starting with Conan 1.35](https://github.com/conan-io/conan/pull/8650) we will be able to use this field, meanwhile, it needs to be commented out.

---

is this the upcoming solution for #3903 ?